control.lua

```diff
@@ -872,11 +873,12 @@ script.on_event({defines.events.on_tick}, function(event)
 			end
 		end
 		-- Add disabled prototypes
-		if #noxy_trees.disabled_match > 0 then
+		if next(noxy_trees.disabled_match) then
 			for e,_ in pairs(game.entity_prototypes) do
-				for k,_ in pairs(noxy_trees.disabled_match) do
-					if e.find(k) then
-						noxy_trees.disabled[e] = true
+				for k,v in pairs(noxy_trees.disabled_match) do
+					if e:find(k) then
+						noxy_trees.disabled[e] = v
+						nx_debug("Disabled tree spawning for prototype: " .. e)
```

Comment:
Can you remove this debug message?

Reply:
OK will prepare a new branch with changes https://github.com/yvesmerle/Noxys_Trees/tree/pullrequest-tsr-2

Reply:
You should be able to just commit the changes to the original branch. Github should then automatically update this PR.

Reply:
OK new variant is prepared here.